### PR TITLE
doc: Fix absolute/relative URL terminology

### PR DESCRIPTION
**Pull request by ddeboer**

### CODE REVIEW

File: doc/ref/tags/tag_url.rst [hunk 1]
```diff
@@ -4,35 +4,44 @@
...
+Returns the URL “/admin/edit/42?foo=bar”.
 
-Please note that the best way to generate the url of a page (resource) is to use::
+Please note that the best way to generate the URL of a page (resource) is to use::
 
    {{ m.rsc[myid].page_url }}
 
-There is one magic argument: ``use_absolute_url``. When this argument is added then the generated url contains the
-protocol, hostname and port of the website (as seen by the user agent)::
+Generate relative URLs
```
Comment (arjan) [inline doc/ref/tags/tag_url.rst]: Shouldn't that read: "generating absolute URLs"?

Comment (ddeboer) [inline doc/ref/tags/tag_url.rst]: Lol, of course it should! Thanks for reading along. :wink:

**Comment by mworrell** [comment]
Good correction.  👍 

Merging and cherry-picking into master

**Comment by mworrell** [comment]
Done, thanks again! 🍰

**Pull request merged**